bigquery/google/cloud/bigquery/job.py

```diff
@@ -913,31 +957,27 @@ def _build_resource(self):
             'tableId': table.table_id,
         } for table in self.sources]
 
-        resource = {
+        configuration = self._configuration.to_api_repr()
+        configuration['sourceTables'] = source_refs
+        configuration['destinationTable'] = {
```

Comment:
Sanity check: Is there a case where `configuration['destinationTable']` would already be set? (Not saying there is.)

Reply:
There certainly is:
```py
job = client.get_job("a-job-id")
```
Will set both job.destination and job._configuration['destinationTable'].  Is that a concern?

Reply:
The current behavior is the one I'd expect as a user since `destination_table` is not defined as a property of the `CopyJobConfig` class. The value set on the `CopyJob` should take precedence.